LEAF_Nexus/sources/Group.php

```diff
@@ -804,6 +834,9 @@ public function togglePermission($groupID, $categoryID, $UID, $permissionType)
      */
     public function addPermission($groupID, $categoryID, $UID, $permissionType)
     {
+        if(!is_numeric($groupID) || !is_numeric($UID)){
+            return null;
```

Comment:
There shouldn't be a check against `$UID` being numeric, since it's not always guaranteed that it will be numeric, it depends on the class.

Reply:
Looks like you missed this one.